### PR TITLE
update zh-translation content/zh/docs/tasks/configure-pod-container/configure-runasusername.md

### PR DESCRIPTION
**Pull request by gorda**
Ref: [k8smeetup#3114](https://github.com/k8smeetup/website-tasks/issues/3114)

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-runasusername.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+
+<!--
+This page shows how to enable and use the `RunAsUserName` feature for pods and containers that will run on Windows nodes. This feature is meant to be the Windows equivalent of the Linux-specific `runAsUser` feature, allowing users to run the container entrypoints with a different username that their default ones.
+-->
+
+本页显示将展示如何为运行在 Windows 节点上的 pod 和容器启用并使用 `RunAsUserName` 功能。此功能旨在成为 Windows 版的 `runAsUser`（Linux），此功能允许用户使用与默认用户名不同的用户名运行容器 entrypoint。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-runasusername.md]:
```suggestion
本页展示如何为运行在 Windows 节点上的 pod 和容器启用并使用 `RunAsUserName` 功能。此功能旨在成为 Windows 版的 `runAsUser`（Linux），允许用户使用与默认用户名不同的用户名运行容器 entrypoint。
```

File: content/zh/docs/tasks/configure-pod-container/configure-runasusername.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+<!--
+## Windows Username limitations
+
+In order to use this feature, the value set in the `runAsUserName` field must be a valid username. It must have the following format: `DOMAIN\USER`, where `DOMAIN\` is optional. Windows user names are case insensitive. Additionally, there are some restrictions regarding the `DOMAIN` and `USER`:
+-->
+
+## Windows Username 的局限性
+
+想要使用此功能，在 `runAsUserName` 字段中设置的值必须是有效的用户名。它必须是 `DOMAIN\USER` 这种格式，其中 `DOMAIN\` 是可选的。Windows 用户名不区分大小写。此外，这儿还有一些关于 `DOMAIN` 和 `USER` 的限制：
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-runasusername.md]:
```suggestion
想要使用此功能，在 `runAsUserName` 字段中设置的值必须是有效的用户名。它必须是 `DOMAIN\USER` 这种格式，其中 `DOMAIN\` 是可选的。Windows 用户名不区分大小写。此外关于 `DOMAIN` 和 `USER` 还有一些限制：
```

**Comment by gorda** [comment]
@tengqm I fix it. thanks for your review!

**Comment by xichengliudui** [comment]
/approve
@gorda good job!

**Pull request merged**